content/en/docs/reference/config/istio.operator.v1alpha1/index.html

```diff
@@ -120,18 +120,6 @@ <h2 id="IstioOperatorSpec">IstioOperatorSpec</h2>
 <p>Identify the revision this installation is associated with.
 This option is currently experimental.</p>
 
-</td>
-<td>
-No
-</td>
-</tr>
-<tr id="IstioOperatorSpec-defaultRevision">
```

Comment:
I think this is generated and refered to the IstioOperator spec in the api repo, which should not edit here. IIRC this is not implemented and only has the api doc.